templates/travel/elements/_input.css

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /**
+ * input
+ *
+ * Create a stylized input element. Derived from the basscss .input class with
+ * some non-compatible changes.
+ *
+ * Modifier Classes:
+ * - input-dark: style the input as a semi-transparent black overlay with
+ *    white text.
+ * - input-big: increace the element padding
+ */
+.travel-input {
+  padding: var(--input-padding-y) var(--input-padding-x);
+  line-height: var(--input-line-height);
+  background-color: var(--white);
+  border-width: var(--border-width);
+  border-style: solid;
+  color: currentColor;
+}
+
+.travel-input:not(.border) {
+  border-color: transparent;
+}
+
+.travel-input-dark {
+  background: color(#000 a(0.2));
+}
+
+.travel-input-clear {
+  color: var(--white);
+  background: none;
+  border-color: rgba(255, 255, 255, .4);
+}
+
+.travel-input-big {
+  padding: var(--input-big-padding-y) var(--input-big-padding-x);
+}
+
+.travel-input::-webkit-input-placeholder {
+  color: var(--gray);
+}
+
+.travel-input::-moz-placeholder {
+  color: var(--gray);
+  opacity: 1;
+}
+
+.travel-input:-ms-input-placeholder {
+  color: var(--gray);
+}
+
+.travel-input-dark::-webkit-input-placeholder {
+  color: currentColor;
+  opacity: 0.75;
+}
+
+.travel-input-dark::-moz-placeholder {
```

Comment:
club the rules into one, why create so much redundant selectors? (also sometimes the gulp processing does it for you , check if your output has these rules created for you. if you just did ::placeholder)